test/upload-api.test.js

```diff
@@ -137,4 +151,24 @@ test('POST / client can upload a file and list it', async t => {
   }, {
     interval: 100,
   })
+
+  // Check metrics were updated
+  beforeW3AccumulatedSize && await pWaitFor(async () => {
+    const afterOperationW3Metrics = await getW3Metrics(t)
+    const afterW3AccumulatedSize = afterOperationW3Metrics.find(row => row.name === W3_METRICS_NAMES.STORE_ADD_ACCUM_SIZE)
+
+    return afterW3AccumulatedSize?.value > beforeW3AccumulatedSize.value
```

Comment:
Please can we test the real value after storing _multiple_ times?

Reply:
Goal here was to create a balance to get to a point where we do not have flaky tests. When this integration tests run in the PR scope is actually fine. However, they also run in staging. Making such assumption in staging can be problematic because of parallel engineer testing something in staging or when we hook up other tests in staging that write stuff there (for example, for w3infra)...

We can make a conditional more explicit. If is staging, just accept `>` and if it is PR expect real value different.

What do you think?

Reply:
Did this in commit